app/controllers/orgs/lti_configurations_controller.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Orgs
+  class LtiConfigurationsController < Orgs::Controller
+    before_action :ensure_lti_launch_flipper_is_enabled
+    before_action :ensure_current_lti_configuration, except: %i[new create]
```

Comment:
Should this instead be the inverse?
```suggestion
    before_action :ensure_current_lti_configuration, only: :show
```